server/models.py

```diff
@@ -285,6 +285,13 @@ class Assignment(Model):
     files = db.Column(JsonBlob)  # JSON object mapping filenames to contents
     course = db.relationship("Course", backref="assignments")
 
+    publish_tags = [tag.title() if tag != 'private' else 'Hidden' for tag in GRADE_TAGS]
+    grade_visibility = db.Column(db.String(255), nullable=False, default="Hidden")
```

Comment:
How does this handle multiple tags? Will the default value pass the check constraint?

I think this should be nullable (for ease of migration) - but also because it makes sense to have no tags visible. 

An example of how we support a list of elements is available in the OAuth models (see the `Client` class)